### DISCUSSION TITLE
Disable Export permission from role in list of dashboards

### DISCUSSION THREAD

**Discussion by Narendra678** [apache/superset#20069] [Q&A / Help]
Hi Team,

I need to disable export option in superset, May i know which role(permissions) i have to remove.

![image](https://user-images.githubusercontent.com/88739186/168248149-93be6c11-05cf-40d0-b4e7-e2f498a4efba.png)

Regards,
Naren

**Comment by diegomedina248** [comment]
Hi,

You can configure these permissions per role, going to `Settings -> List Roles` and selecting the role you want to edit.
I believe the permission you want to change/remove is `can export on Dashboard`.

**Reply by Narendra678** [reply]
Thanks for reply.

I did not see "can export on Dashboard" in admin or Gamma roles. Can i have exact permission name please?

Regards,
Naren
